core/ui/src/views/settings/SettingsSmartHost.vue

```diff
@@ -0,0 +1,395 @@
+      this.error.test_smarthost = false;
+      const isValidationOk = this.validateConfigureModule();
+      if (!isValidationOk) {
+        return;
+      }
+      this.loading.setSmarthost = true;
+      
+      const taskAction = "set-smarthost";
+
+      // register to task completion
+      this.$root.$once(
+        taskAction + "-completed",
+        this.setSmarthostCompleted
+      );
+      // register to task error
+      this.$root.$once(
+        taskAction+"-aborted",
+        this.setSmarthostAborted
+      );
+
+      this.$root.$once(
+        taskAction + "-validation-failed",
+        this.setSmarthostValidationFailed
+      );
+
+      // register to task completion
+      this.$root.$once(
+        taskAction + "-completed",
+        this.setSmarthostCompleted
+      );
+
+      const res = await to(
+        this.createClusterTask({
+          action: taskAction,
+          data: {
+            host: this.enabled ? this.host : "",
+            username: this.enabled ? this.username : "",
+            password: this.enabled ? this.password : "",
+            port: this.enabled ? parseInt(this.port) : 587,
+            tls: this.enabled ? this.tls : true,
+            tls_verify: this.enabled ? this.tls_verify : true,
+            enabled: this.enabled
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: true,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.setSmarthost = this.getErrorMessage(err);
+        this.loading.setSmarthost = false;
+        return;
+      }
+    },
+    setSmarthostValidationFailed(validationErrors) {
+      this.loading.setSmarthost = false;
+      let focusAlreadySet = false;
+
+      for (const validationError of validationErrors) {
+        const param = validationError.parameter;
+
+        // set i18n error message
+        this.error[param] = this.getI18nStringWithFallback(
+          "smarthost." + validationError.error,
+          "error." + validationError.error
+        );
+
+        if (!focusAlreadySet) {
+          this.focusElement(param);
+          focusAlreadySet = true;
+        }
+      }
+    },
+    setSmarthostCompleted() {
+      this.loading.setSmarthost = false;
+
+      //// don't do this at every task completed
+      this.getSmarthost();
+    },
+    validateConfigureModule() {
+      for (const key of Object.keys(this.error)) {
+        this.error[key] = "";
+      }
+
+      let isValidationOk = true;
+      if (!this.host) {
+        this.error.host = this.getI18nStringWithFallback("smarthost.required");
+        if (isValidationOk) {
+          this.focusElement("host");
+        }
+        isValidationOk = false;
+      }
+      return isValidationOk
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import "../../styles/carbon-utils";
+
+.break-word {
```

Comment:
This CSS class is not used, we can remove it

Reply:
done